text/0000-test-framework-binnacle.md

```diff
@@ -0,0 +1,357 @@
+- Start Date: 2020-01-07
+- Tracking: (leave this empty)
+- RFC PR: [kadalu/rfcs#0000](https://github.com/kadalu/rfcs/pull/0000)
+
+# Test Framework - Binnacle
+
+## Authors
+
+- Aravinda Vishwanathapura <aravinda@kadalu.io>
+- Amar Tumballi <amar@kadalu.io>
```

Comment:
my name not required.. its all your idea, and thinking. I just helped in 'agree'ing to this.